build.gradle

```diff
@@ -31,9 +31,11 @@ repositories {
 
 ext {
     springCloudServiceBrokerVersion = '3.6.1'
+    springWso2DependenciesVersion = '5.3.36-wso2v1'
```

Comment:
why can't this just be called `springFrameworkVersion`?  wso2 is part of the version number now, but might not be in the future

Reply:
Agree that Including wso2 in variable name is not a good idea. Renamed it to the suggested.

Fyi, other libs, like spring-core, in spring framework are on 5.3.31 (implicit, from springboot). The version specified by this variable is currently applicable only for 3 of those libs.